Formula/gcc.rb

```diff
@@ -98,27 +97,28 @@ def install
       --with-mpfr=#{Formula["mpfr"].opt_prefix}
       --with-mpc=#{Formula["libmpc"].opt_prefix}
       --with-isl=#{Formula["isl"].opt_prefix}
-      --with-system-zlib
       --with-pkgversion=#{pkgversion}
       --with-bugurl=#{tap.issues_url}
     ]
 
-    # Xcode 10 dropped 32-bit support
-    args << "--disable-multilib" if DevelopmentTools.clang_build_version >= 1000
+    on_macos do
+      args << "--build=#{cpu}-apple-darwin#{OS.kernel_version.major}"
+      args << "--with-system-zlib"
```

Comment:
Don't you want this too on Linux, since you have `uses_from_macos "zlib"`?

Reply:
If I use that flag, it will link against ubuntu's system zlib.
I want it to link against `/home/linuxbrew/.linuxbrew/lib/libz.so.1`, so I have to remove that flag on Linux.

I did not do a test build with that flags on Linux, but this is how I understand that flag's logic.